app/routes/erddap/utils.js

```diff
@@ -0,0 +1,56 @@
+const ids = [
```

Comment:
does this really need to be hardcoded? or can we derive it from the available data in the dataset?